tests/testapps/AddressBookSample/AddressBookSample.vsimporter/AddressBookSample-WinStore10/Package.appxmanifest

```diff
@@ -1,29 +1,49 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
-  <Identity Name="34906fc5-08c8-4bad-b833-46a91bb7cf95" Publisher="islandwood" Version="1.0.0.0" />
-  <mp:PhoneIdentity PhoneProductId="34906fc5-08c8-4bad-b833-46a91bb7cf95" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
+<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="f7a26016-402a-44f3-ae0e-3f41c17a4180"
+    Publisher="CN=islandwood"
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="f7a26016-402a-44f3-ae0e-3f41c17a4180" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
   <Properties>
     <DisplayName>AddressBookSample</DisplayName>
-    <PublisherDisplayName>islandwood</PublisherDisplayName>
+    <PublisherDisplayName>bbowman</PublisherDisplayName>
     <Logo>Assets\StoreLogo.png</Logo>
   </Properties>
+
   <Dependencies>
     <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
   </Dependencies>
+
   <Resources>
     <Resource Language="x-generate" />
   </Resources>
+
   <Applications>
-    <Application Id="App" Executable="$targetnametoken$.exe" EntryPoint="AddressBookSample.App">
-      <uap:VisualElements DisplayName="AddressBookSample" Square150x150Logo="Assets\Square150x150Logo.png" Square44x44Logo="Assets\Square44x44Logo.png" Description="AddressBookSample" BackgroundColor="transparent">
-        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png">
-        </uap:DefaultTile>
+    <Application Id="App"
+      Executable="$targetnametoken$.exe"
+      EntryPoint="AddressBookSample.App">
+      <uap:VisualElements
+        DisplayName="AddressBookSample"
+        Square150x150Logo="Assets\Square150x150Logo.png"
+        Square44x44Logo="Assets\Square44x44Logo.png"
+        Description="AddressBookSample"
+        BackgroundColor="transparent">
+        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png"/>
         <uap:SplashScreen Image="Assets\SplashScreen.png" />
       </uap:VisualElements>
     </Application>
   </Applications>
+
   <Capabilities>
     <Capability Name="internetClient" />
-    <uap:Capability Name="contacts" />
```

Comment:
We should really be better about automatically adding these capabilities. +1 to me.

---
In reply to: [102806885](https://github.com/Microsoft/WinObjC/pull/2056#discussion_r102806885) [](ancestors = 102806885)